webpack/ForemanTasks/Components/TasksTable/SubTasksPage.js

```diff
@@ -15,7 +15,8 @@ export const SubTasksPage = props => {
       { caption: __('Sub tasks') },
     ],
   });
-  const createHeader = actionName => actionName ? sprintf(__('Sub tasks of %s'), actionName) : __('Sub tasks');
+  const createHeader = actionName =>
+    actionName ? sprintf(__('Sub tasks of %s'), actionName) : __('Sub tasks');
```

Comment:
Is this needed? From what I've seen the breadcrumbs are always `Tasks > $action_name > Sub tasks`

Reply:
In some tasks the action is empty. But we can do a fall back like we did here instead: https://github.com/adamruzicka/foreman-tasks/blob/c6bf988bb25fc473d44d4ee8ca6d18353a168f6c/webpack/ForemanTasks/Components/TasksTable/TasksTableSelectors.js#L39

Reply:
I see, then it is fine the way it is